src/auth-service/controllers/create-user.js

```diff
@@ -281,6 +282,83 @@ const createUser = {
       });
     }
   },
+  deleteMobileUserData: async (req, res) => {
+    try {
+      const { query, body } = req;
+      let { tenant } = query;
+      let responseFromDeleteAppData;
+      logText("We are deleting the app data.....");
+      const { email, phoneNumber } = req.query;
+      
+       if (!email && !phoneNumber) {
+         return res.status(400).json({ errors: [{ msg: 'Either email or phoneNumber is required' }] });
```

Comment:
Could we maintain the standard response format, as other endpoints? Just to maintain uniformity?

**More like:**
_{success: true or false, message: "the general message", errors: {message: "details of the problem"}_

Reply:
Thanks @Baalmart 
I updated these